README.md

```diff
@@ -135,33 +135,22 @@ You can send metrics, histograms, or trace data from your application to the Wav
 [Create a DirectConfiguration](#option-2-sending-data-via-direct-ingestion) to send data directly to a Wavefront service.
   
 ### Option 1: Sending Data via the Wavefront Proxy
-Depending on the data you wish to send to Wavefront (metrics, distributions (histograms) and/or spans), enable the relevant ports on the proxy and initialize the proxy sender.
 
 ```go
 import (
     wavefront "github.com/wavefronthq/wavefront-sdk-go/senders"
 )
 
 func main() {
-    proxyCfg := &wavefront.ProxyConfiguration {
-        Host : "proxyHostname or proxyIPAddress",
-
-        // At least one port should be set below.
-        MetricsPort         : 2878,    // set this (typically 2878) to send metrics
-        DistributionPort    : 2878,   // set this (typically 2878) to send distributions
-        TracingPort         : 30000, // set this to send tracing spans. the same port as the customTracingListenerPorts configured in the wavefront proxy
-        FlushIntervalSeconds: 10,   // flush the buffer periodically, defaults to 5 seconds.
-    }
-
-    sender, err := wavefront.NewProxySender(proxyCfg)
+    sender, err := wavefront.NewSender("http://localhost")
```

Comment:
Add a comment to give users a clue about where metrics, histograms, and spans go. Based on the URL I would presume everything is sent to localhost:80. Something like:
`// By default for Proxy endpoints, metrics and histograms are sent to port 2878 and spans are sent to port 30001`

Reply:
Done.